pkgs/test_api/lib/src/backend/state.dart

```diff
@@ -64,16 +64,12 @@ final class Status {
   final String name;
 
   factory Status.parse(String name) {
-    switch (name) {
-      case 'pending':
-        return Status.pending;
-      case 'running':
-        return Status.running;
-      case 'complete':
-        return Status.complete;
-      default:
-        throw ArgumentError('Invalid status name "$name".');
-    }
+    return switch (name) {
```

Comment:
This could just be `Status.values.fromName` or similar, right?

Reply:
We could refactor these into enums. I don't think it's likely to be breaking in practice. @jakemac53 - do you think there is any risk?

Reply:
#2028